### DISCUSSION TITLE
Resets when reading certain NES & Famicom games

### DISCUSSION THREAD

**Discussion by RottenToTheGore** [sanni/cartreader#528] [Need help]
Using HW4 with swapable cart adapters. Some games I've had trouble with that just needed a good cleaning or reseating a couple times, but I'm pretty sure there's something on the hardware side that I must have messed up. 

The games in question are...

> Dragon Quest III - Famicom
Karate Champ - NES USA
Mega Man - NES USA

The game will be read, often finding a database match. To be sure I'm choosing the correct game I scroll through the list using the newest update to the nes.ino that removed auto selecting https://github.com/sanni/cartreader/issues/521#issuecomment-1250358956

I also make sure to check NESCartDB just to be sure there's nothing off there too. Go to read NES rom, and once it gets to the progress bar it resets back to the main menu of the reader, leaving me with a 0kb ROM and no log file in the game folder. I don't have the main log file with me but as a mockup it looks like this...

> Saving to NES/ROM/GameName/123/...
[*

And then it resets. 

So far I've reflowed the solder on the NES adapter, not the Famicom one just yet. I know the next part would be to bust out the multimeter and check out the connections on the adapter and the main board itself, but I thought if I posted what's happening and with what games, maybe there's a way to pinpoint where the trouble could be coming from.

Thanks!

**Comment by RottenToTheGore** [comment]
Mapped out all the points on the cartridge port adapter to where they go to on the board to test connections, everything passed the multimeter test.

**Comment by sanni** [comment]
You can try removing the following line from Cart_Reader.ino

![8584848484848oll](https://user-images.githubusercontent.com/19169494/193156132-1caf6b91-c61e-43ae-9b19-472beed64898.jpg)

The Cart Reader will then use the old NES dumping function. After selecting "NES/Famicom" you will have to manually set up the mapper with the "Change Mapper" menu selection and finally you can dump the ROM with "Read iNES Rom".

If this also fails maybe try a different SD card. Or if you have [enabled additional consoles](https://github.com/sanni/cartreader/blob/master/Cart_Reader/Cart_Reader.ino#L93) in Cart_Reader.ino try with the default selection.
